src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
-  return (
```

Comment:
- [ ] Kindly change the `App` component to a class-based component so as to meet the requirements of this project.